bot/src/main/java/com/dongtronic/diabot/util/Patterns.kt

```diff
@@ -25,6 +25,13 @@ object Patterns {
      * Matches suspected BG values with units
      */
     val unitBgPattern = Regex("""(?<value>\d{1,3}(?:[.,]\d+)?) ?(?<unit>mmol|mg)""")
+
+    /**
+     * Matches special regex characters. May be used with `replace` to sanitize inputs
+     * for use in a regex.
+     */
+    val regexSanitizerPattern = Regex("[-.\\+*?\\[^\\]$(){}=!<>|:\\\\]")
```

Comment:
Going with the assumption that this is needed (and that `Regex.escape` wouldn't work):

I would use a raw string literal, to avoid excessive escaping:
```kotlin
val regexSanitizerPattern = Regex("""[-.\+*?\[^\]$(){}=!<>|:\\]""")
```

Also I don't think the `\+` needs to be escaped:
```kotlin
val regexSanitizerPattern = Regex("""[-.+*?\[^\]$(){}=!<>|:\\]""")
```